core/src/main/java/de/danoeh/antennapod/core/feed/FeedItem.java

```diff
@@ -388,6 +391,22 @@ public boolean hasChapters() {
         return hasChapters;
     }
 
+    public void setAutoDownload(boolean autoDownload) {
+        this.autoDownload = autoDownload;
+    }
+
+    public boolean getAutoDownload() {
+        return this.autoDownload;
+    }
+
+    public boolean isAutoDownloadable() {
+        return this.hasMedia() &&
+                false == this.getMedia().isPlaying() &&
```

Comment:
Ha. It's certainly more readable (I even noticed that as I was typing my comment). Thanks for bringing it to my attention.